src/models/MyAccount.ts

```diff
@@ -154,12 +154,25 @@ export default class MyAccount {
     return bibs.reduce((bibDataMap: BibDataMapType, bibFields) => {
       const { isResearch, isNyplOwned } =
         this.getResearchAndOwnership(bibFields)
-      const title = bibFields.title
+      const title = `${bibFields.title}${
+        bibFields.author && isNyplOwned ? ` / ${bibFields.author}` : ""
```

Comment:
I'm guessing that the date + "- author" is a data problem?

![Screenshot 2024-06-26 at 2 48 45 PM](https://github.com/NYPL/research-catalog/assets/1280564/aee3879f-8773-4706-b9cc-7b7a588e84d2)

Reply:
Yes. I think some author fields have the years the author is alive for, and also the word author apparently.